src/css/swagger.css

```diff
@@ -297,6 +304,19 @@
   display: none;
 }
 
+::part(section-navbar-item section-navbar-path) {
+  padding-left: 1.5rem;
+}
+
+::part(section-navbar-tag) {
+  padding-left: 1.25rem;
+}
+
+::part(section-navbar-operations-top) {
+  justify-content: flex-start;
+  margin-left: -6px;
+}
+
```

Comment:
[question] I imagine this is fine, but the preview url doesn't show the API docs. Do you have a screenshot of what was changed here?

[future] We should figure out how to get the preview urls working showing the API docs, so that we can verify for regression etc.

Reply:
I'll add a preview page to this PR

Reply:
https://deploy-preview-198--docs-ui.netlify.app/api#servers